packages/launchpad/cypress/e2e/choose-a-browser.cy.ts

```diff
@@ -117,7 +117,8 @@ describe('Choose a Browser Page', () => {
       }).as('launchProject')
 
       cy.get('@launchButton').click()
-      cy.contains('button', 'Opening E2E Testing in Chrome')
+      // TODO: Relies on ctx.emitter.toLaunchpad which doesn't refresh AUT
```

Comment:
Is this work being tracked somewhere?

Reply:
No, I'll create an issue and update this comment. I talked with @tgriesser a bit about this and tried to find out why this wasn't working but had no luck